components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -4801,159 +4801,80 @@ func TestListProjects(t *testing.T) {
 			require.NoError(t, err)
 			assert.Empty(t, ps)
 		}},
-		{"when two projects (custom and chef-managed) exist, returns them", func(t *testing.T) {
+		{"when multiple projects exist, filter based on projects lists", func(t *testing.T) {
 			ctx := context.Background()
-			insertTestProject(t, db, "foo", "my foo project", storage.ChefManaged)
-			insertTestProject(t, db, "bar", "my bar project", storage.Custom)
+			p1 := insertTestProject(t, db, "foo", "my foo project", storage.ChefManaged)
+			p2 := insertTestProject(t, db, "bar", "my bar project", storage.Custom)
```

Comment:
PR Bonus: use the output of the helper function here to eliminate a lot of code in declaring the expectations just below.